schematic/store/synapse.py

```diff
@@ -603,10 +604,91 @@ def upload_project_manifests_to_synapse(self, projectId: str) -> List[str]:
 
+    def upload_annotated_project_manifests_to_synapse(self, projectId:str, path_to_json_ld: str) -> List[str]:
+        '''
+        Purpose:
+            For all manifests in a project, upload them as a table and add annotations manifest csv.
+            Assumes the manifest is already present as a CSV in a dataset in the project.
+
+        '''
+
+        sg = SchemaGenerator(path_to_json_ld)
+        manifests = []
+        manifest_loaded = []
+        datasets = self.getStorageDatasetsInProject(projectId)
+        for (datasetId, datasetName) in datasets:
+            # encode information about the manifest in a simple list (so that R clients can unpack it)
+            # eventually can serialize differently
+
+            manifest = ((datasetId, datasetName), ("", ""), ("", ""))
+
+            manifest_info = self.getDatasetManifest(datasetId, downloadFile=True)
+
+            if manifest_info:
+                manifest_id = manifest_info["properties"]["id"]
+                manifest_name = manifest_info["properties"]["name"]
+                manifest_path = manifest_info["path"]
+                manifest_df = load_df(manifest_path)
+                #manifest_syn_id = self.annotate_upload_manifest_table(manifest_df, datasetId, manifest_path)
+                manifest_syn_id = self.associateMetadataWithFiles(sg, manifest_path, datasetId, manifest_record_type='table')
+                manifest_loaded.append(datasetName)
+        return
+
+
+    def move_entities_to_new_project(self, projectId: str, newProjectId: str, returnEntities: bool = False):
+        """
+        For each manifest csv in a project, look for all the entitiy ids that are associated.
+        Look up the entitiy in the files, move the entity to new project.
+        """
+
+        manifests = []
+        manifest_loaded = []
+        datasets = self.getStorageDatasetsInProject(projectId)
+        for (datasetId, datasetName) in datasets:
+            # encode information about the manifest in a simple list (so that R clients can unpack it)
+            # eventually can serialize differently
+
+            manifest = ((datasetId, datasetName), ("", ""), ("", ""))
+
+            manifest_info = self.getDatasetManifest(datasetId, downloadFile=True)
+            if manifest_info:
+                manifest_id = manifest_info["properties"]["id"]
+                manifest_name = manifest_info["properties"]["name"]
+                manifest_path = manifest_info["path"]
+                manifest_df = load_df(manifest_path)
+
+                annotation_entities = self.storageFileviewTable[
+                        (self.storageFileviewTable['id'].isin(manifest_df['entityId']))
+                        & (self.storageFileviewTable['type'] == 'folder')
+                    ]['id']
+
+                if returnEntities:
+                    for entityId in annotation_entities:
+                        self.syn.move(entityId, datasetId)
+                        pass
```

Comment:
What is this pass statement for?

Reply:
I'm leaving them in until after the dry runs with HTAN are complete, they just helped with debugging